src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx

```diff
@@ -215,7 +215,10 @@ export default function DownloadDialog({
         )
         setGameInstallInfo(gameInstallInfo)
 
-        if (gameInstallInfo && 'languages' in gameInstallInfo.manifest) {
+        if (
```

Comment:
needed to add this condition otherwise if a game was shown as installed on legendary installed.json file but was being marked as not available, clicking on install button was causing it to fail.
Now the install screen shows normally.